packages/cli/__tests__/zosjobs/__system__/download/__scripts__/download-output/download.sh

```diff
@@ -16,6 +16,9 @@ then
     exit $RC
 fi
 
+# Echo the job ID for further use
+echo "Submitted job ID: $JOBID"
+
 # Loop until the job goes to the output queue
 until [ $ATTEMPTS -gt 0 ]
```

Comment:
This goes back to the previous comment about `-gt` v. `-le` 😋

Reply:
i think im going to revert back to gt for consistency (in that other file)